src/sentry/tasks/store.py

```diff
@@ -228,15 +230,18 @@ def _do_symbolicate_event(cache_key, start_time, event_id, symbolicate_task, dat
     symbolication_start_time = time()
 
     with sentry_sdk.start_span(op="tasks.store.symbolicate_event.symbolication") as span:
-        span.set_data("symbolicaton_function", symbolication_function.__name__)
+        span.set_data("symbolication_function", symbolication_function_name)
         with metrics.timer(
             "tasks.store.symbolicate_event.symbolication",
-            tags={"symbolication_function": symbolication_function.__name__},
+            tags={"symbolication_function": symbolication_function_name},
         ):
             while True:
                 try:
+                    if symbolication_function is None:
+                        break
```

Comment:
Since we'll still be timing this (via `metrics.timer`, and also in `sentry_sdk.start_span`), this can skew the metrics/APM. 
Wdyt about either adding a new tag so we can filter those bad things out, or e.g. setting "symbolication_function_name" to some predefined value?

Reply:
avoiding emitting this metric altogether now